src/main/spec/internal-billing-api-spec.yaml

```diff
@@ -83,6 +83,52 @@ paths:
           $ref: "../../../spec/error-responses.yaml#/$defs/InternalServerError"
       tags:
         - internalBilling
+  /internal/rpc/remittance/reset_billable_usage_remittance:
+    put:
+      summary: Update billable usage remittance records.
```

Comment:
Please add some details of what it means to 'reset' billable usage.

Reply:
Done